I built it manually using 

```
$ nix develop .#osm2pgsq

$ phases="${prePhases[*]:-} unpackPhase patchPhase" genericBuild
$ phases="${preConfigurePhases[*]:-} configurePhase ${preBuildPhases[*]:-} buildPhase" genericBuild

```

and can confirm that `install-gen` target doesn't exist

```
make install-gen
make: *** No rule to make target 'install-gen'.  Stop.
```

I also checked the changes in source code and I couldn't find anything relevant.

--- a/pkgs/tools/misc/osm2pgsql/default.nix
+++ b/pkgs/tools/misc/osm2pgsql/default.nix
@@ -7,7 +7,6 @@
 , bzip2
 , zlib
 , boost
-, cimg
 , postgresql
 , python3
 , withLuaJIT ? false
@@ -15,6 +14,7 @@
 , luajit
 , libosmium
 , nlohmann_json
+, opencv
 , potrace
 , protozero
 , testers
@@ -22,13 +22,13 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "osm2pgsql";
-  version = "1.9.2";
+  version = "1.10.0";
 
   src = fetchFromGitHub {
     owner = "osm2pgsql-dev";
     repo = "osm2pgsql";
     rev = finalAttrs.version;
-    hash = "sha256-RzJpaOEpgKm2IN6CK2Z67CUG0WU2ELvCpGhdQehjGKU=";
+    hash = "sha256-IFAQ7iA37QXnWOSxUjh9EW7ss85k0h948JGuuUcpr5w=";
   };
 
   postPatch = ''
@@ -41,11 +41,11 @@ stdenv.mkDerivation (finalAttrs: {
   buildInputs = [
     boost
     bzip2
-    cimg
     expat
     fmt
     libosmium
     nlohmann_json
+    opencv
     postgresql
     potrace
     proj

